test/unit/spec/PasswordReset_spec.js

```diff
@@ -40,18 +40,23 @@ function (Q, _, $, OktaAuth, LoginUtil, Util, PasswordResetForm, Beacon, Expect,
       excludeUsername: true
     };
 
-    if (settings && settings.policyComplexity) {
+    if (settings && (settings.policyComplexity || settings.policyAge)) {
       passwordResetResponse = deepClone(resPasswordResetWithComplexity);
       var responsePolicy = passwordResetResponse.response._embedded.policy;
 
-      var key = settings.policyComplexity;
-      if (key === 'all') {
+      if (settings.policyComplexity === 'all') {
         responsePolicy.complexity = policyComplexityDefaults;
       }
-      else {
+      else if (settings.policyComplexity) {
+        var key = settings.policyComplexity;
         responsePolicy.complexity[key] = policyComplexityDefaults[key];
       }
       delete settings.policyComplexity;
+
+      if (settings.policyAge === 'history') {
+        responsePolicy.age.historyCount = 7;
```

Comment:
nit: move the ```7``` to a variable named ```historyCountDefault```, just so we have the default values all in the same place